pkgs/os-specific/linux/kernel-headers/default.nix

```diff
@@ -16,7 +16,7 @@ let
     # We do this so we have a build->build, not build->host, C compiler.
     depsBuildBuild = [ buildPackages.stdenv.cc ];
     # `elf-header` is null when libc provides `elf.h`.
-    nativeBuildInputs = [ perl elf-header ];
+    nativeBuildInputs = [ perl rsync elf-header ];
```

Comment:
The new `make install_headers` installs the headers using rsync (to copy only `.h` files).